commands/utils/packagehandlers/packagehandlers_test.go

```diff
@@ -222,11 +224,18 @@ func TestYarnPackageHandler_UpdateDependency(t *testing.T) {
 				VulnerabilityOrViolationRow: formats.VulnerabilityOrViolationRow{Technology: coreutils.Yarn, ImpactedDependencyName: "minimist"},
 			}, fixSupported: true,
 		},
+		{
+			vulnDetails: &utils.VulnerabilityDetails{
+				SuggestedFixedVersion:       "1.2.6",
+				IsDirectDependency:          true,
+				VulnerabilityOrViolationRow: formats.VulnerabilityOrViolationRow{Technology: coreutils.Yarn, ImpactedDependencyName: "minimist"},
+			}, fixSupported: true,
+		},
 	}
-	for _, test := range testcases {
-		t.Run(test.vulnDetails.ImpactedDependencyName+" direct:"+strconv.FormatBool(test.vulnDetails.IsDirectDependency), func(t *testing.T) {
+	for idx, test := range testcases {
+		t.Run(test.vulnDetails.ImpactedDependencyName+" direct:"+strconv.FormatBool(test.vulnDetails.IsDirectDependency)+",v"+strconv.Itoa(idx), func(t *testing.T) {
 			testDataDir := getTestDataDir(t, test.vulnDetails.IsDirectDependency)
-			cleanup := createTempDirAndChDir(t, testDataDir, coreutils.Yarn)
+			cleanup := createTempDirAndChDir(t, testDataDir, coreutils.Yarn, strconv.Itoa(idx))
```

Comment:
why we need strconv.Itoa(idx)?

Reply:
in line 234 for creating an informative name for the test
in line since createTempDirAndChDir takes extraArgs as ...string in its current state and not accepting int. I can change the signature to accept forth argument(s) as int but it will be less generic code